products/zoningtaxlots/bash/02_build.sh

```diff
@@ -3,16 +3,15 @@ source bash/config.sh
 
 echo "Run build"
 
-run_sql_file sql/create_priority.sql &
+run_sql_file sql/create_priority.sql
```

Comment:
removing making these background processing - not horribly long to run and it gets around `set -e` when things go wrong (case in point I deleted the file area_zoningdistrict_create.sql without removing the `run_sql_file` command belowand the job ran just fine)